code/modules/mob/living/simple_animal/hostile/tree.dm

```diff
@@ -27,13 +27,28 @@
 	emote_taunt = list("growls")
 	taunt_chance = 20
 
-	//Space carp aren't affected by atmos.
-	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	//Space carp aren't affected by atmos. //Space Carps no. But trees Yes
+	atmos_requirements = list("min_oxy" = 2, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	unsuitable_atmos_damage = 5
 	minbodytemp = 0
+	maxbodytemp = 400
 
 	faction = list("hostile")
 	var/drop_type = /obj/item/stack/sheet/mineral/wood
 
+/mob/living/simple_animal/hostile/tree/Life()
+	..()
+	//Thank'e Phil  for the simplification
+	if(istype(src.loc, /turf/simulated))
+		var/turf/simulated/T = src.loc
+		if(T.air)
+			var/co2 = T.air.carbon_dioxide
+			if(co2 > 0)
+				if(prob(25))
+					var/amt = min(co2, 9)
+					T.air.carbon_dioxide -= amt
+					T.air.oxygen += amt/3*2

```

Comment:
Alright, Ill change it to atmos_spawn_air but what im saying is that the turf would be activate anyway due to the CO2 In it already and if its not the why whould the Oxygen derived from the inactive CO2 Be active.